cmake/Embed.cmake

```diff
@@ -22,6 +22,10 @@
 # THE SOFTWARE.
 #####################################################################################
 
+if(DEFINED embed_wrap_string)
+    return()
+endif()
```

Comment:
Would it be better to use `include_guard(GLOBAL)` or `include_guard(DIRECTORY)` instead?